Resources/Prototypes/_Lavaland/Stacks/Materials/materials.yml

```
@@ -1,48 +1,48 @@
 - type: stack
   id: GoldOreUnprocessed
-  name: unprocessed gold ore
+  name: native gold
   icon: { sprite: /Textures/Objects/Materials/ore.rsi, state: gold }
```

YAMLlint (1.35.1)

[error] too many spaces inside braces

(braces)